### PR TITLE
added protection-domain-nodes test, changed tests to check several protection domains

### PR DESCRIPTION
**Pull request by tikitavi**
!!!do not merge until merging fuel-charms pull request #4

### CODE REVIEW

File: juju/fuel/functions [hunk 1]
```diff
@@ -178,15 +178,25 @@ function configure_cluster() {
     shift
   done
 
-  sleep 15
+  wait_and_check
+}
+
+function add_computes() {
+  for i in `echo $1 | sed 's/,/ /g'`; do
```
Comment (alexey-mr) [inline juju/fuel/functions]: this logic is already exist in deploy_node_service function.
you can either just use it directly (and remove add_compute) or keep add_compute for semantic and just recall deploy_node_service with comma separated list of compute nodes.

File: juju/fuel/deploy-fuel.sh [hunk 2]
```diff
@@ -6,18 +6,19 @@ my_dir="$(dirname $my_file)"
...
 
 # prepare fuel master
 prepare_fuel_master 0
 
 # 3+1 cluster with default parameters:
-#   zero-padding=false
-#   TODO: add more parameters and check their default value
 configure_cluster mode 1 primary-controller 1 compute 2,3
+# Adding compute node to check that it will get to the same protection domain (parameter protection-domain-nodes is 100 by default)
+configure_cluster mode 1 primary-controller 1 compute 4
```
Comment (alexey-mr) [inline juju/fuel/deploy-fuel.sh]: configure_cluster mode 1 primary-controller 1 compute 2,3,4

File: juju/fuel/deploy-fuel.sh [hunk 3]
```diff
@@ -44,9 +48,11 @@ set_fuel_options cached-storage-pools='sp2'
 set_fuel_options rfcache-devices=$rfcache_paths
 set_fuel_options sds-on-controller='false'
 configure_cluster mode 1 primary-controller 1 compute 2,3,4
+configure_cluster mode 1 primary-controller 1 compute 5,6,7
```
Comment (alexey-mr) [inline juju/fuel/deploy-fuel.sh]: configure_cluster mode 1 primary-controller 1 compute 2,3,4,5,6,7

File: juju/fuel/deploy-fuel.sh (hunk 2 above)
Comment (tikitavi) [inline juju/fuel/deploy-fuel.sh]: But why? It seems, that it is working the same way.

**Pull request merged**